samples/extensions/khr/externalmemory/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2021 The Khronos Group Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Example only works with NVIDIA backend, so we look for a CUDA-capable GPU
+# on the system before adding the tests.
+
+# FindCUDA module.
+find_package(CUDA)
```

Comment:
I'm not using FindCUDA (only forgot to remove the comment because at first I used it and then I saw that it was deprecated), or is `find_package(CUDA)` also invoking FindCUDA?